paths/electron/starting-with-electron/03-git-repository.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Creating a Git Repository
+permalink: /electron/starting-with-electron/init-git-repository
+next-page: /electron/starting-with-electron/create-app-files
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  We have a great start of an Electron app. But, it's not being tracked with version control. Now, it's time to track this repository with Git, include a .gitignore file that fits this project, and push this up to a remote on GitHub.
```

Comment:
Hmmm good point, @hectorsector. I'm not sure if we should delete them or not. I think it's the recommended tool for creating a boilerplate. I think we should either delete them, or add info in the 'tell me why' explaining the additional files.